### ISSUE TITLE
'SpotifyiOS.h' file not found

### ISSUE THREAD

**Issue by zachlyoung** [cjam/react-native-spotify-remote#223]
Hey @cjam, I'm getting some Xcode build errors while trying to get the example up and running.

Line 10 of RNSpotifyRemoteError: `'SpotifyiOS.h' file not found`

I followed all of steps outline in the README.md and I also ran `yarn submodules` in the root directory. Here's a [video](https://youtu.be/2M4NTcsIeoc) of all the steps I took (I rotated the Spotify client secret shown in the video!)

Any idea of what I did wrong here?

**Comment by toddriley** [comment]
There are a couple issues with running the example app. The first is that line 14 of the Podfile assumes the dependency is in a parent directory instead of inside `node_modules`. 

https://github.com/cjam/react-native-spotify-remote/blob/7c13126ec6ce5e990a83aa730b65382ce709e37a/example/ios/Podfile#L14

If you update this line to be the following:

```
pod 'RNSpotifyRemote', :path => '../node_modules/react-native-spotify-remote'
```

Then you'll get the same error because the `react-native-spotify-remote` dependency is missing from `package.json`. But if you run:

`yarn add react-native-spotify-remote`

Then that error will go away.

**Comment by cjam** [comment]
Hey, sorry for the delay.  I'll have to take a gander at trying to run the example app again.  As for @toddriley's comment, it is true that it points upward two directories.  This was so that the example app would use the latest code within the library itself, so it was intended but perhaps it has stopped working as intended.

**Comment by johra** [comment]
Also getting the same problem, @cjam an update on example would be greatly appreciated. Trying to get the project running using Expo and EAS build. Almost done appart from this and also the linking of xcframework instead of framework.

**Comment by chen-rn** [comment]
> Also getting the same problem, @cjam an update on example would be greatly appreciated. Trying to get the project running using Expo and EAS build. Almost done appart from this and also the linking of xcframework instead of framework.

Hey! did you manage to get this working with a Managed Expo dev client?

**Comment by johra** [comment]
> 

Yes, but I hade to make a special expo-plugin for it, see here: https://github.com/EkornEvent/Tempofy/blob/master/spotify-plugin/index.js

**Comment by chen-rn** [comment]
Oh that looks fantastic!

**Comment by Wulfheart** [comment]
> > 
> 
> Yes, but I hade to make a special expo-plugin for it, see here: https://github.com/EkornEvent/Tempofy/blob/master/spotify-plugin/index.js

So does it work without XCode?

**Comment by johra** [comment]
Yes that is correct

**Comment by Wulfheart** [comment]
Oh this is cool. Then I don’t have to buy a MacBook!
